genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga

```diff
@@ -2,7 +2,7 @@
     "a_galaxy_workflow": "true",
     "annotation": "",
     "format-version": "0.1",
-    "name": "COVID-19: Variant Calling Lofreq",
+    "name": "COVID-19 - Genomics [4] Variant Calling Lofreq",
```

Comment:
Please don't change the name. The ` - Genomics [4]` part doesn't make any sense outside of this repo.

Reply:
Maybe I change all the other workflow names too than ?

Reply:
👍 I think that's a good idea

Reply:
I will add genomics and cheminformatics maybe as tag, to have that as visual distinction if these workflows will get buried in a galaxy instance? Or are these short names of the workflows self explanatory enough?

Reply:
The tags sounds good!

Reply:
@mvdbeek done!